setup.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+"""
+Caring Caribou
+==============
+- A friendly automotive security exploration tool, initiated as part of the research project HEAVENS (HEAling Vulnerabilities to ENhance Software Security and Safety), now a stand-alone project.
+- A zero-knowledge tool that can be dropped onto an automotive network and collect information regarding what services and vulnerabilities exist.
+"""
+
+from setuptools import find_packages, setup
+
+version = "0.4dev"
+dl_version = "master" if "dev" in version else "v{}".format(version)
+
+print(r"""-----------------------------------
+ Installing Caring Caribou version {0}
+-----------------------------------
+""".format(version))
+
+setup(
+    name="caringcaribou",
+    version=version,
+    author="Kasper Karlsson",
+    author_email="hej@cc.py",
```

Comment:
I have no email address which I would like to include here. Sorry for my lack of knowledge in python packaging, but is there a best practice for this scenario? E.g. completely leave the field out, or something like `no-email@cc.invalid`?

Reply:
I commented `author_email` in the last push, don't actually know whether one is required for package distribution.